src/main/java/com/cinema/shop/validators/PasswordValidator.java

```diff
@@ -0,0 +1,14 @@
+package com.cinema.shop.validators;
+
+import com.cinema.shop.model.dto.UserRequestDto;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class PasswordValidator implements ConstraintValidator<ValidPassword, UserRequestDto> {
+
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return userRequestDto.getPassword().equals(userRequestDto.getRepeatPassword());
```

Comment:
does it work if  userRequestDto.getPassword() == null?

Reply:
I thought if there is a @NotNull annotation above the password than repeat password can't be null

Reply:
this validation may execute first